scripts/run-ci-python-typecheck.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
Not a blocker, but where is this file used?

Reply:
I use this manually to check that the type check passes. Technically it can be added to CI, but I left that out for now.

Reply:
Sounds good.